src/gam/__init__.py

```diff
@@ -7020,6 +7020,9 @@ def _splitSchemaNameDotFieldName(sn_fn, fnRequired=True):
             else:
                 body[up][schemaName][fieldName] = sys.argv[i]
             i += 1
+        elif myarg == 'verifynotinvitable':
+            body['verifyNotInvitable'] = True
```

Comment:
It's pretty ugly that were over-extending body here. I'd prefer to see us return a tuple of verify_not_invitable and body.